### DISCUSSION TITLE
so-test fails in ver. 2.3.21

### DISCUSSION THREAD

**Discussion by jpl33** [Security-Onion-Solutions/securityonion#2452] [Unsupported Versions] [answered]
Command 'docker pull --disable-content-trust=false docker.io/securityonion/so-tcpreplay:2.3.21' failed with return code: 1

but 
command ' sudo docker pull --disable-content-trust=false docker.io/securityonion/so-tcpreplay:2.3.10'
succeeds and downloads the 2.3.10 docker.
seems like some trouble with docker.io?

**Comment by jertel** [comment, marked as answer]
Tracking this issue in #2463.

**Reply by circuitswitch** [reply]
I am following along with the You Tube Security Essentials.  
securityonion installed in Eval mode.
so-test command failed.
Output of so-test attached.
[soTestFail.txt](https://github.com/Security-Onion-Solutions/securityonion/files/5830653/soTestFail.txt)

**Reply by circuitswitch** [reply]
Is there a work around while this is fixed?

**Comment by M507** [comment]
@circuitswitch 
Install the latest update:
```
sudo soup;sudo soup
```
